apps/meteor/ee/server/models/raw/LivechatPriority.ts

```diff
@@ -24,4 +24,12 @@ export class LivechatPriorityRaw extends BaseRaw<ILivechatPriority> implements I
 
 		return this.findOne(query, options);
 	}
+
+	async canResetPriorities(): Promise<boolean> {
+		return Boolean(await this.findOne({ dirty: true }));
```

Comment:
if you just want to check if a record exists, pls use a `projection` to fetch only the `_id` field.